app/views/payments/openPayments.scala.html

```diff
@@ -55,20 +44,18 @@ <h1 class="heading-xlarge">@messages("openPayments.heading")</h1>
           }
       </div>
 
-      @model.hasDirectDebit match {
-          case Some(hasDD) => {
-              @if(!hasDD) {
-                  <div id="processing-time">
-                      <p>@messages("openPayments.processingTime")</p>
-                  </div>
-              }
-          }
-          case None => {
-              <div id="processing-time-old">
-                  <p>@messages("openPayments.processingTimeOld")</p>
-              </div>
-          }
-      }
+        <div id="processing-time">
+            <p>@messages("openPayments.processingTime")</p>
+        </div>
+
+        <h2 id="what-you-owe-missing" class="heading-small">@messages("openPayments.whatOweHelp.heading")</h2>
+            <p>@messages("openPayments.incorrect")
+                <a target="_blank" href="https://www.gov.uk/vat-corrections">@messages("openPayments.correctErrors")</a>.
+            </p>
+            <p>@messages("payment.afterSubmitted") @Messages("payment.stillMake")
+                <a target="_blank" href="@appConfig.unauthenticatedPaymentsUrl" id="vatPaymentsLink">
+                    @messages("payment.stillMakeLink")<span class="visuallyhidden"> @messages("payment.evenIfNotShown")</span></a>.
+            </p>
```

Comment:
A bit nit-picky, but could you re-format this section of code please

Reply:
Done